content/maxwell3_fdem/emi_tutorial/mutualinductance.rst

```diff
@@ -0,0 +1,136 @@
+
+.. math::
+        \Phi_2 = \frac{\mu_0 I_1}{4\pi} \oint \left ( \oint \frac{dl_1}{\lvert \mathbf{r} - \mathbf{r'}\rvert^2} \right ) \cdot dl_2.
+        :label: phi23
+
+We can then write the mutual inductance between two loops as:
+
+.. math::
+        M_{12} = \frac{\mu_0}{4\pi} \oint \oint \frac{dl_1 \cdot dl_2}{\lvert \mathbf{r} - \mathbf{r'}\rvert^2}.
+        :label: m12
+
+There are a few significant things about Equation :eq:`m12`:
+
+.. note::
+    - :math:`M_{12}` depends purely on geometry, such as the size, shape, and relative positions of the two loops
+    - This expression doesn't change if we look at the flux in the first loop due to the second loop, meaning that :math:`M_{12} = M_{21}`. Therefore, following reciprocity staifies from Eq. :eq:`EMF2`
+
+    .. math::
+        M_{12} = \frac{\Phi_2}{I_1} = \frac{\Phi_1}{I_2}.
+        :label: EMF2_1
+
+So,  by solving Equation :eq:`m12` for the three mutual inductances for a three-loop system, we can analytically solve Equation :eq:`l2` and determine the data :math:`H^s / H^p` over different targets, using differing frequencies, loop orientations, and loop separations. This can provide meaningful understanding about three-loop systems (such as the EM-31 and Resolve systems) and their data.
+
+Effects of coupling
+-------------------
+
+.. figure:: ./images/LoopOrientations.png
+    :figwidth: 30%
+    :align: right
+    :name: LoopOrientations
+
+    The orientation of the loops can be changed by adjusting the inclination `I` and the declination `D`.
+
+The primary and secondary magnetic field are measured at the Loop3 (Rx), and we set our EM datum as :math:`\frac{H^s}{H^p}`. If the primary and secondary field are in opposing directions at the receiver, we consider this to be a negative datum. If they are in the same direction, the datum is positive. Hence, for instance :numref:`Concepts_3loops_only_M` has negative datum. Accordingly, :math:`A` in Eq. :eq:`l2`, which defines geometric coupling should be positive. :numref:`LoopOrientations` shows the orientation of the loop with inclination (I) and declination (D). Considering I and D of the Loop2, :numref:`Concepts_3loops_only_M` can be considered as I=0 and D=90 degree. For this case, we consider a loop-loop EM survey with the geometry of EM-31 (3.66m Tx-Rx offset; CITE), and the body (Loop2) is embedded 3 m below the surface. :numref:`EMdataI0D90` shows complex response function (Plot1), easting line at 0 m-northing (Plot2), plan maps of real and imaginary part of the EM data with the loop-loop EM survey when :math:`\alpha` =0.31. Plot2 clearly shows how coupling is changing depending on relative location and orientation of three loop system. Plot3 and Plot4 illustrate plan map of EM data we could possibly obtain by acquring multiplie lines. Following considers different orientation of the body (Loop2), which will generate different coupling effects on the measured EM data.
```

Comment:
showing the fields lines for a couple scenarios (one positive, one negative, one null-coupled would be beneficial here